packages/opentelemetry-node/package.json

```diff
@@ -24,11 +24,13 @@
   },
   "peerDependencies": {
     "@opentelemetry/api": "1.x",
+    "@opentelemetry/core": "1.x",
     "@opentelemetry/sdk-trace-base": "1.x",
     "@opentelemetry/semantic-conventions": "1.x"
   },
   "devDependencies": {
     "@opentelemetry/api": "^1.2.0",
+    "@opentelemetry/core": "^1.7.0",
```

Comment:
m: I _think_ we can only add this to peerDependencies and leave it as a transitive dependency in dev deps. but also OK to add it explicitly, I guess! If we want to leave it here, we should _also_ add `@opentelemetry/semantic-conventions` as devDependency for symmetry!